src/styles/cylc/_job.scss

```diff
@@ -0,0 +1,112 @@
+/* css namespace for job element */
+$cjob: ".c-job svg.job rect";
+
+
+#{$cjob} {
+    /* if no job status display nothing */
+    fill: transparent;
+    stroke: transparent;
+}
+
+
+.job_theme--normal {
+    $teal: rgb(125,207,212);
+    $blue: rgb(106,164,241);
+    $green: rgb(81,175,81);
+    $red: rgb(207,72,72);
+    $pink: rgb(190,106,192);
+
+    #{$cjob} {
+        &.submitted {
+            fill: $teal;
+            stroke: $teal;
+        }
+
+        &.running {
+            fill: $blue;
+            stroke: $blue;
+        }
+
+        &.succeeded {
+            fill: $green;
+            stroke: $green;
+        }
+
+        &.failed {
+            fill: $red;
+            stroke: $red;
+        }
+
+        &.submit-failed {
+            fill: $pink;
+            stroke: $pink;
+        }
+    }
+}
+
+
+.job_theme--greyscale {
+    $light: rgb(208,208,208);
+    $medium: rgb(133,133,133);
+    $dark: rgb(0,0,0);
+
+    #{$cjob} {
+        &.submitted {
+            fill: transparent;
+            stroke: $light;
+        }
+
+        &.running {
+            fill: $light;
+            stroke: $light;
+        }
+
+        &.succeeded {
+            fill: $medium;
+            stroke: $medium;
+        }
+
+        &.failed {
+            fill: $dark;
+            stroke: $dark;
+        }
+
+        &.submit-failed {
+            fill: transparent;
+            stroke: $dark;
+        }
+    }
+}
+
+.job_theme--colour_blind {
+    $grey: rgb(152,152,152);
+    $success: rgb(108,218,255);
+    $error: rgb(146,0,0);
```

Comment:
For future development I think it would for clarity be better to name these variables to give an indication as to the resulting colours rather than the corresponding job state category, which can be seen from the maps in ``#{$cjob}`` anyway (for the ``greyscale`` theme it is obvious from context they are all a shade of grey of lightness as per the name).

So perhaps you could use ``cb-blue`` & ``cb-red`` instead of ``success`` & ``error``, respectively?

Reply:
This was kinda on purpose on the basis of it being a colour-blind theme.

Reply:
Are the variable names ever exposed to the users? I assumed that only developers would encounter them, & if so I think my point stands (though it is minor & isn't a blocker in any way). But if the names are somehow processed & displayed to users then definitely, your point makes a lot of sense.